base_site/index.html

```diff
@@ -140,15 +165,15 @@ <h1 class="heading-style-h1 text-color-gray">Technical Documentation</h1>
           <div class="container">
             <div class="padding-section-small">
               <div class="component_grid">
-                <a href="mailto: qcsupport@quantinuum.com" class="button_link is-support w-inline-block">
-                  <div class="button_link_image-wrap"><img src="images/support.svg" loading="lazy" alt="" class="image-fit-contain"></div>
+                <a href="#" class="button_link is-support w-inline-block">
```

Comment:
```suggestion
                <a href="mailto: qcsupport@quantinuum.com" class="button_link is-support w-inline-block">
```

I'm guessing this should continue to have the `mailto:` link in. Another one to feed back to Luke to keep in the original, I guess?